### PR TITLE
Pr correct typos

### PR DESCRIPTION
**Pull request by PeaaZee**
Hello,

I began going through the new documentation, and corrected a few points on the way that caught my eye. I mainly corrected typos and little grammar mistakes. I also made slight changes to the original wording when I thought the original wording could be misunderstanding. I tried to put these in different commits so that you can sort them easier.

I then began adding metric units to wire gauge, as that is the unit far more common here in the EU (in germany at least).

Unfortunately I don't have more time and you wanted to check all PRs until Dec 21st. That's why I wanted this to get out there.

Feel free to cherry pick to your liking!

Regards

### CODE REVIEW

File: official/about.md [hunk 1]
```diff
@@ -31,22 +31,24 @@ There are a few reasons Voron does not and will not officially support active ch
...
 
-Watercooling is not supported because of the risk associated with it, leaks and mains voltage don't play well with each other. You need a separate cable chain to properly run the tubing which would not fit inside the machine (XY is mostly the issue, but there is an issue with the Z also).  Adding a water pump and a chamber heater will easily push it near a 15a current draw, as internal testing has determined a we need approximately 700w of heating power to warm the chamber to 75C (per multiple ansys simulations).
+The true recommended chamber temp for PEEK is 130C, for True Nylon or PC (not hybrids or blends) is 100C, below this you will see delamination. This basicaly equates to putting your printer in an oven, you'd want the least amount possible of the motion system in the "hot zone" and a ton of insulation. VORON is currently not designed for this, and it is not planned to be.
```
Comment (jdlongenecker) [inline official/about.md:36]: I like this rewording, generally watercooling is niche enough it probably doesn't need a direct comment. However, I would remove the comment about Nylons and PC and focus more on PEEK/PEI--we do have users printing a variety of Nylons and PC materials.

File: official/about.md [hunk 2]
```diff
@@ -31,22 +31,24 @@ There are a few reasons Voron does not and will not officially support active ch
...
 
-The true recommended chamber temp for PEEK is 130C, for True Nylon or PC (not hybrids or blends) is 100C, below this you will see delamination. This basicaly equates to putting your printer in an oven, youd want the least amount possible of the motion system in the "hot zone" and a ton of insulation. VORON is currently not designed for this, and it is not planned to be.
+This machine is intended such that it can be built with "garden tools", chamber heaters violate this, and add unnecessary risk to the end user. We recommend you purchase an industrial printer designed for this purpose if you intend to print with these materials so you don't end up with a fire on your hands.
```
Comment (jdlongenecker) [inline official/about.md:38]: this is great, just remove everything after "if you intend to print with these materials"

File: official/about.md [hunk 3]
```diff
@@ -31,22 +31,24 @@ There are a few reasons Voron does not and will not officially support active ch
...
 
-This machine is intended such that it can be built with "garden tools", chamber heaters violate this, and add unnecessary risk to the end user. We recommend you purchase an industrial printer designed for this purpose if you intend to print with these materials so you dont end up with a fire on your hands.
+### Watercooling
```
Comment (jdlongenecker) [inline official/about.md:40]: I see what you did in this PR now--reorganizing the paragraphs. Let's completely remove this paragraph about watercooling and the 15A current draw/Ansys simulations. The previous documentation this was from is completely and comically wrong.

**Review by jdlongenecker — COMMENTED** [review]
This is looking great! Some of my comments are more general and may be better suited to another PR on these items :)

File: official/about.md [hunk 4]
```diff
@@ -2,23 +2,23 @@
 
 All Voron printers are built using CoreXY or CoreXZ configurations.  No Cartesian motion systems to be found.  The following reviews some of common features of Voron printers.
 
-## Voron Does
+## What a Voron does
```
[the remark this answers is not in the document]
Comment (bdbell) [reply official/about.md:5]: Those title lines are originally mine.  I was trying to be amusing as I couldn't think of better names.

**Comment by bdbell** [comment]
Merging as per discussion.

**Pull request merged**